src/coreclr/vm/corelib.h

```diff
@@ -285,9 +302,11 @@ DEFINE_METHOD(CURRENCY,             DECIMAL_CTOR,           .ctor,
 DEFINE_CLASS(DATE_TIME,             System,                 DateTime)
 DEFINE_METHOD(DATE_TIME,            LONG_CTOR,              .ctor,                      IM_Long_RetVoid)
 
+BEGIN_ILLINK_FEATURE_SWITCH(System.Runtime.InteropServices.BuiltInComInterop.IsSupported, true, true)
 #ifdef FEATURE_COMINTEROP
 DEFINE_CLASS(DATE_TIME_OFFSET,      System,                 DateTimeOffset)
```

Comment:
`DATE_TIME_OFFSET` looks unused. Can you delete it from here instead? (Also, I would be surprising to have DateTimeOffset-specific code related to COM interop.)